P1/Project_Code/frontend/src/components/Media_Cards/MediaCard.jsx

```diff
@@ -21,16 +21,28 @@ const MediaCard = ({ movie, type }) => {
                 ?(
                     type.toLowerCase() == "trending"
                     ?
-                        (   <div style={{ width: "440px", height: "280px", backgroundImage: `url(${imgSrc})`, backgroundPosition: "center", backgroundRepeat: "no-repeat", backgroundSize: "cover"}}>
-                                <h2 className="heading-M" style={{ color: "white", marginTop: "15rem", fontWeight: "bolder"}}>{movie.title}</h2>
+                        (   <div style={{ borderRadius: "20px", width: "540px", height: "280px", backgroundImage: `url(${imgSrc})`, backgroundPosition: "center", backgroundRepeat: "no-repeat", backgroundSize: "cover"}}>
```

Comment:
You should move the styling logic into its own file. Create a file in Media_Cards folder called MediaCard.styles.js